components/inputs/README.md

```diff
@@ -2,14 +2,13 @@
 
 There are various input components available:
 
-- [Text](#text-inputs)
-- [Text Areas](#text-areas)
-- [Search](#search-inputs)
-- [Select Lists](#select-lists)
-- [Checkboxes](#checkboxes)
-- [Radio Buttons](#radio-buttons)
-- [Date](#date-inputs)
-- [Date-Time](#date-time-inputs)
```

Comment:
Nit: I'd be OK with just having the MD files inline and named the same as the `.js` file or in `docs/` (shorter)?